### PR TITLE
Rename config yaml keys camel case

### PR DESCRIPTION
**Pull request by spangenberg**
Keys were pascal cased, now camel cased.

### CODE REVIEW

File: installer/pkg/config/aws/config.go [hunk 1]
```diff
@@ -2,19 +2,19 @@ package aws
...
+	AssetsS3BucketName        string    `yaml:"assetsS3BucketName,omitempty"`
+	AutoScalingGroupExtraTags string    `yaml:"autoScalingGroupExtraTags,omitempty"`
+	EC2AMIOverride            string    `yaml:"ec2AMIOverride,omitempty"`
+	Etcd                      component `yaml:"etcd,omitempty"`
+	External                  external  `yaml:"external,omitempty"`
+	ExtraTags                 string    `yaml:"extraTags,omitempty"`
+	InstallerRole             string    `yaml:"installerRole,omitempty"`
+	Master                    component `yaml:"master,omitempty"`
+	PrivateEndpoints          bool      `yaml:"privateEndpoints,omitempty"`
+	Profile                   string    `yaml:"profile,omitempty"`
```
Comment (mxinden) [inline installer/pkg/config/aws/config.go]: seems to be wrongly indented.

File: installer/pkg/config/gcp/config.go [hunk 2]
```diff
@@ -2,11 +2,11 @@ package gcp
...
 // Config defines the GCP configuraiton for a cluster.
 type Config struct {
-	ConfigVersion            string    `yaml:"ConfigVersion,omitempty"`
-	Etcd                     component `yaml:"Etcd,omitempty"`
-	ExtGoogleManagedZoneName string    `yaml:"ExtGoogleManagedZoneName,omitempty"`
-	Master                   component `yaml:"Master,omitempty"`
-	Region                   string    `yaml:"Region,omitempty"`
-	SSHKey                   string    `yaml:"SSHKey,omitempty"`
-	Worker                   component `yaml:"Worker,omitempty"`
+	ConfigVersion            string    `yaml:"configVersion,omitempty"`
```
Comment (mxinden) [inline installer/pkg/config/gcp/config.go]: go fmt

File: installer/pkg/config/types.go [hunk 3]
```diff
@@ -11,99 +11,99 @@ import (
...
+	Admin             admin            `yaml:"admin"`
+	Azure             azure.Config     `yaml:"azure,omitempty"`
+	BaseDomain        string           `yaml:"baseDomain"`
+	CA                ca               `yaml:"ca"`
+	ContainerLinux    containerLinux   `yaml:"containerLinux"`
+	CustomCAPEMList   string           `yaml:"customCAPEMList"`
+	DDNS              ddns             `yaml:"ddns"`
+	DNSName           string           `yaml:"dnsName"`
+	Etcd              etcd             `yaml:"etcd"`
+	GCP               gcp.Config       `yaml:"gcp,omitempty"`
```
Comment (mxinden) [inline installer/pkg/config/types.go]: go fmt

**Review by mxinden — CHANGES_REQUESTED** [review]
Can you adjust the tests config as well, otherwise your tests will fail: https://github.com/coreos/tectonic-installer/blob/ut2-integration/tests/smoke/aws/vars/aws.basic.yaml

**Comment by spangenberg** [comment]
retest this

**Comment by mxinden** [comment]
ok to test

**Comment by mxinden** [comment]
@spangenberg would you mind adjusting https://github.com/coreos/tectonic-installer/blob/master/tests/rspec/lib/config_file.rb#L21 and all following lines as well to the new casing?

File: installer/pkg/config/types.go (hunk 3 above)
Comment (mxinden) [reply installer/pkg/config/types.go]: Seems to be only in my browser, not in `raw` mode.

**Review by mxinden — APPROVED** [review]

**Pull request merged**